client/src/Login.tsx

```diff
@@ -17,155 +18,197 @@ function Login() {
   // Don't autoload user into app if they navigated from app
   let { state } = useLocation();
   const autoLoad = state === null ? true : false;
-
+  const usingElectron = window.electronAPI === undefined ? false : true;
+  // Input fields
   const [accessKey, setAccessKey] = useState("");
   const [secretKey, setSecretKey] = useState("");
   const [clusterName, setClusterName] = useState("");
   const [regionName, setRegionName] = useState("");
   // Toggling Password visiblity
   const [hideEye, toggleEye] = useState(true);
   const [hidePassword, togglePassword] = useState(true);
-  // Whether to show the loading screen
+  // Toggle loading screen
   const [isLoading, setLoading] = useState(false);
 
   // Only fire on initial render
   // Executes twice because of React.Strict
   useEffect(() => {
-    // On mount
     // Don't load user in if they just logged out
-    if(autoLoad !== false) {
+    if (autoLoad !== false) {
+      setLoading(true);
       loginUser(true);
     } else {
-      setLoading(true);
-      // Then load config
-      window.electronAPI.getConfig();
-    }
-    // Response after trying to update local cred/config files
-    window.electronAPI.onConfigResp('onConfigResp', (event: any, data: any) => {
-      console.log("From server:", data)
-      setLoading(false);
-      // The fourth datapoint is whether the kube config was set properly
-      // Only attempt to log the user in if kube config was created
-      if(data[4]) {
-        loginUser(false);
-      } else {
-        alert(badCredentials);
+      // Only send request to electron if using electron
+      if (window.electronAPI) {
+        setLoading(true);
+        window.electronAPI.getConfig();
       }
-    })
-    // Why is this executing twice?
-    // Response after parsing local cred/config files
-    window.electronAPI.onSendConfig('onSendConfig', (event: any, data: [string, string, string, string]) => {
-      console.log("Parsed from local files:", data);
-      setLoading(false);
-      // Update each input field
-      setAccessKey(data[0]);
-      setSecretKey(data[1]);
-      setClusterName(data[2]);
-      setRegionName(data[3]);
-    })
+    }
+
+    // Handles electron's response for setting config files
+    if (usingElectron) {
+      window.electronAPI.onConfigResp('onConfigResp', (event: any, data: any) => {
+        // console.log("From server:", data)
```

Comment:
This looks leftover. Is there a reason to put it into our primary codebase? If so, probably explain.